src/components/app/data/utils.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Check if system maintenance alert is open, based on configuration.
+ * @param {Object} config
+ * @returns {boolean}
+ */
+export function isSystemMaintenanceAlertOpen(config) {
+  if (!config) {
+    return false;
+  }
+  const isEnabledWithMessage = (
+    config.IS_MAINTENANCE_ALERT_ENABLED && config.MAINTENANCE_ALERT_MESSAGE
+  );
+  if (!isEnabledWithMessage) {
+    return false;
+  }
+  const startTimestamp = config.MAINTENANCE_ALERT_START_TIMESTAMP;
+  if (startTimestamp) {
+    return new Date() > new Date(startTimestamp);
+  }
```

Comment:
[nit/suggestion] I am wondering if its worth using dayjs here for the comparison with [difference](https://day.js.org/docs/en/display/difference)

Reply:
Sure, given its fairly low effort to replace this with `dayjs`; that said, generally, I feel we should minimize the amount of refactoring of existing logic as we start integrating with more of the existing UI components. In this case, this was copy/pasted from another location within the application but otherwise unchanged. Given your suggestion is only impacting a handful of lines, though, I've opted to migrate to `dayjs` here but using its `isAfter` function.